physutils/io.py

```diff
@@ -8,13 +8,124 @@
 import os.path as op
 
 import numpy as np
+from bids import BIDSLayout
 from loguru import logger
 
 from physutils import physio
 
 EXPECTED = ["data", "fs", "history", "metadata"]
 
 
+def load_from_bids(
```

Comment:
Would it be possible to add the parameter `recording=None` ? To reason is that according to the current version of the BIDS spec physio data with different sampling rate need to be saved under different file names using the `recording` entity to differentiate them apart.

Reply:
And recording should be added when you call `layout.get`

Reply:
Got it, done!